README.md

```diff
@@ -0,0 +1,10 @@
+Bykr
+====
+
+Installation
```

Comment:
I'm actually working on this right now. I thought that documentation should not be its own PR because it's not a user/app/admin function. Should documentation be its own PR, then?